R/get_national_data.R

```diff
@@ -0,0 +1,155 @@
+#' Get national-level data for countries globally, sourced from the ECDC or WHO.
+#' 
+#' @description Gets raw data using the source-specific function. Includes ISO country codes. 
+#' Then adds columns which were missing from the raw data (calculating cumulative counts from new dailies and vice versa),
+#' cleans and sanitises further. Adds rows and columns of NA values so that data is in a standard format. 
+#' 
+#' @param country Character A string specifying the country to get data from. Not case dependent. Defaults to all countries.
+#' Name should be the English name.
+#' @param totals Boolean. If TRUE, returns totalled data per country up to today's date. If FALSE, returns the full dataset stratified by date and country.
+#' @param source Character A string specifying the data source: "WHO", or "ECDC". Not case dependent. Defaults to ECDC.
```

Comment:
at some point it would be good to have a check what the default should be - I assume its still ECDC but no real idea